backend/src/interface/persistent_data.rs

```diff
@@ -149,8 +149,8 @@ pub trait RequestDataAccess {
     async fn get_lines(&self, canteen_id: Uuid) -> Result<Vec<Line>>;
     /// Returns the meal related to all the params.
     async fn get_meal(&self, id: Uuid, line_id: Uuid, date: Date) -> Result<Option<Meal>>;
-    /// Returns all meals related to all the params.
-    async fn get_meals(&self, line_id: Uuid, date: Date) -> Result<Vec<Meal>>;
+    /// Returns all meals related to all the params. Null is returned when there is not any information available yet.
+    async fn get_meals(&self, line_id: Uuid, date: Date) -> Result<Option<Vec<Meal>>>;
```

Comment:
What if the lines are also not available yet? Is it possible that the meal plan contains no lines?

Reply:
Either we already checked the mealplan, then we have data. When we did not find any lines (or lines without any meals) we will just take that as "the line/canteen is closed".
When we did not check the website for mealplans yet (because it is too far in the future), we will just return `None` marking it as "we have no datat".

To implement that we may just store the last date, we have data for.

Is this ok or am i missing sth?